onnxruntime/test/providers/cpu/nn/batch_norm_op_test.cc

```diff
@@ -720,7 +720,7 @@ TEST(BatchNormTest, BatchNorm2d_fp16) {
   test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kTensorrtExecutionProvider});
 }
 
-// flacky test - disabled for now
+// TODO fix flaky test (https://msdata.visualstudio.com/Vienna/_workitems/edit/596949)
 TEST(BatchNormTest, DISABLED_ForwardTrainingTest) {
```

Comment:
Is it ok to link internal task in github code or should we instead create a github issue for tracking this?

Reply:
It is not ok. Here are the guidelines https://docs.opensource.microsoft.com/content/releasing/index.html. 

Remove sensitive assets.
* Remove any reference in the code to internal or confidential information, including internal paths, tools, codenames, proprietary fonts, internal telemetry and email aliases.
* Remove any trademarks or product icons.
* Prepare the code and commit history for publication, including running PoliCheck.

Reply:
removed them here: https://github.com/microsoft/onnxruntime/pull/3643